Makefile.am

```diff
@@ -221,12 +221,10 @@ collectd_SOURCES = \
 	src/daemon/types_list.h \
 	src/daemon/utils_threshold.c \
 	src/daemon/utils_threshold.h
-
-
-collectd_CFLAGS = $(AM_CFLAGS)
-collectd_CPPFLAGS = $(AM_CPPFLAGS)
-collectd_LDFLAGS = -export-dynamic
-collectd_LDADD = \
+libcollectd_la_CPPFLAGS = $(AM_CPPFLAGS) $(LTDLINCL)
+libcollectd_la_CFLAGS = $(AM_CFLAGS)
+libcollectd_la_LDFLAGS = $(COMMON_LDFLAGS)
```

Comment:
What is COMMON_LDFLAGS?

Reply:
That is only set on Windows, and I forgot to remove it when separating out these changes. Removed it now.